keras_cv/bounding_box/converters_test.py

```diff
@@ -107,6 +103,10 @@ def test_converters(self, source, target):
         )
 
     @parameterized.named_parameters(*test_image_ragged)
+    @pytest.mark.skipif(
```

Comment:
I ended up wanting to use something like this a lot, so just with a `@pytest.mark.tf_only`. Then did the reason and skip logic once in [conftest.py](https://github.com/mattdangerw/keras-nlp/blob/60a4f7c99a325a6b63f3444a91ac25e34b1e3fb4/keras_nlp/conftest.py).

Could consider something like that? Or `@pytest.mark.requires_ragged` if you want to be more find grained. Just a bit less code.

Reply:
Done -- thanks for the pointer!